todo_app/templates/index.html

```diff
@@ -7,11 +7,22 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+
+  <form action="/items" method="post">
```

Comment:
You can use url_for in the views too:

```suggestion
  <form action="{{ url_for("add_items") }}" method="post">
```